### ISSUE TITLE
Not able to add mods that have incompatible game versions

### ISSUE THREAD

**Issue by KyleUltimate** [theRookieCoder/ferium#36]
Ex. https://modrinth.com/mod/eating-animation/versions
This only has 1.17.1 listed on modrinth, and when I try to add it to ferium, it spits out this error
![image](https://user-images.githubusercontent.com/68589851/166451228-73d7f096-9366-4466-96d6-ec778ed03f8f.png)
I know there is a `check_game_version` config, but it's not possible to add that if ferium doesn't have the mod stored in config.

**Comment by theRookieCoder** [comment]
Yup I just noticed this when reviewing the code for the previous push. Would adding some flags for disabling these checks to the `add` commands be a good fix?

**Comment by KyleUltimate** [comment]
yep, smth like `--force` `-f`
or more specific one like
`--skip-mod-loader-check`
`--skip-game-version-check`

**Comment by theRookieCoder** [comment]
Fixed in c628c0e8d88740bbaf9e064816957de36f584666
